concierge/service/src/main/java/org/eclipse/ditto/concierge/service/starter/proxy/DefaultEnforcerActorFactory.java

```diff
@@ -115,10 +115,17 @@ public ActorRef startEnforcerActor(final ActorContext context, final ConciergeCo
         final DistributedAcks distributedAcks = DistributedAcks.lookup(actorSystem);
         final LiveSignalPub liveSignalPub = LiveSignalPub.of(context, distributedAcks);
 
+        // creation restriction
+        final CreationRestrictionEnforcer creationRestriction = DefaultCreationRestrictionEnforcer.of(
+                conciergeConfig.getEnforcementConfig().getEntityCreation()
+        );
+        log.debug("Creation restriction: {}", creationRestriction);
```

Comment:
I would suggest to remove this log statement again as it does not really bring any benefit in a productive setup.